rero_ils/config.py

```diff
@@ -1956,30 +1958,67 @@ def _(x):
                     )
                 )
             ),
-            subject=dict(
+            subject_fiction=dict(
```

Comment:
The facets fiction and no_fiction seems identical, facet fileter is missing.

Reply:
Filters are dynamically added using the function [_facet_filter](https://github.com/rero/rero-ils/blob/98dd50a29525a4826b12cdf7fb3b5325eeca5df4/rero_ils/facets.py#L186#L196).